docs/database/mongodb-component.md

```diff
@@ -110,29 +114,18 @@ And then the connection string will be retrieved from the `ConnectionStrings` co
 }
 ```
 
-For more information on how to format this connection string, see [MongoDB: ConnectionString documentation](https://www.mongodb.com/docs/v3.0/reference/connection-string).
-
-### Use configuration providers
-
-The .NET Aspire MongoDB database component supports <xref:Microsoft.Extensions.Configuration?displayProperty=fullName>. It loads the `MySqlConnectorSettings` from configuration files such as _:::no-loc text="appsettings.json":::_ by using the `Aspire:MongoDB:Driver` key. If you have set up your configurations in the `Aspire:MongoDB:Driver` section, you can just call the method without passing any parameter.
-
-The following example shows an _:::no-loc text="appsettings.json":::_ file that configures some of the available options:
+MongoDB Atlas Eexmple:
 
 ```json
 {
-  "Aspire": {
-    "MongoDB": {
-      "Driver": {
-        "ConnectionString": "mongodb://server:port/test",
-        "DisableHealthChecks": false,
-        "HealthCheckTimeout": 10000,
-        "DisableTracing": false
-      },
-    }
+  "ConnectionStrings": {
+    "MongoConnection": "mongodb+srv://username:password@server.mongodb.net/",
   }
 }
 ```
 
+For more information on how to format this connection string, see [MongoDB: ConnectionString documentation](https://www.mongodb.com/docs/v3.0/reference/connection-string).
```

Comment:
Re-add removed section.

```suggestion
For more information on how to format this connection string, see [MongoDB: ConnectionString documentation](https://www.mongodb.com/docs/v3.0/reference/connection-string).

### Use configuration providers

The .NET Aspire MongoDB database component supports <xref:Microsoft.Extensions.Configuration?displayProperty=fullName>. It loads the `MySqlConnectorSettings` from configuration files such as _:::no-loc text="appsettings.json":::_ by using the `Aspire:MongoDB:Driver` key. If you have set up your configurations in the `Aspire:MongoDB:Driver` section, you can just call the method without passing any parameter.

The following example shows an _:::no-loc text="appsettings.json":::_ file that configures some of the available options:

```json
{
  "Aspire": {
    "MongoDB": {
      "Driver": {
        "ConnectionString": "mongodb://server:port/test",
        "DisableHealthChecks": false,
        "HealthCheckTimeout": 10000,
        "DisableTracing": false
      },
    }
  }
```

Reply:
Happy to add this back. However, is it still MySqlConnectorSettings when its MongoDB? I don't know the answer to that.
It also mentions that if you use this section, you don't need to pass any parameter, which I didn't find was the case. .AddMongoClient requires a connection name.

Reply:
I fixed the suggestion, it did have a typo and had incorrect info. Thanks for catching that.